all-dogs.html

```diff
@@ -0,0 +1,157 @@
+        </p>
+          <button type="button" onclick="feeTotal(150.00)">Adopt</button>
+      </article>
+      <article>
+        <img src="./images/jack-card.jpg" onclick="dogInfo('Jake','Beagle ',200.00)" alt="brown and white dog looking up">
+        <h4>Jack</h4>
+        <h5>Cost to Adopt: $200.00</h5>
+        <p>Jack is a Beagle puppy. She is 10 weeks old, neutered, microchipped, vaccinated, playful/happy and ready for a new forever home.
+        </p>
+          <button type="button" onclick="feeTotal(200.00)">Adopt</button>
+      </article>
+      <article>
+        <img src="./images/duffy-card.jpg" onclick="dogInfo('Duffy','Pitbull ',150.00)" alt="white and brown pitbull mix">
+        <h4>Duffy</h4>
+        <h5>Cost to Adopt: $150.00</h5>
+        <p>Duffy is a Pitbull. She is 6 months, neutered, microchipped, vaccinated, house/leash trained and ready for a new forever home.
+        </p>
+          <button type="button" onclick="feeTotal(150.00)">Adopt</button>
+      </article>
+      <article>
+          <img src="./images/lucas-card.jpg" onclick="dogInfo('Lucas ','German Sheppard mix ',150.00)" alt="black German Shppard mix">
+          <h4>Lucas</h4>
+          <h5>Cost to Adopt: $150.00</h5>
+          <p>Lucas is a German Sheppard mix. He is 4 years old, microchipped, vaccinated, house/leash trained and ready for a new forever home.
+          </p>
+            <button type="button" onclick="feeTotal(150.00)">Adopt</button>
+        </article>
+        <article>
+          <img src="./images/jake-card.jpg" onclick="dogInfo('Jake ','Labrador Retriever ',150.00)" alt="black dog looking up">
+          <h4>Jake</h4>
+          <h5>Cost to Adopt: $150.00</h5>
+          <p>Jake is a Labrador. He is 4 years old, neutered, microchipped, vaccinated, house/leash trained and ready for a new forever home.
+          </p>
+            <button type="button" onclick="feeTotal(150.00)">Adopt</button>
+        </article>
+        <article>
+          <img src="./images/angus-card.jpg" onclick="dogInfo('Angus ','Boxer mix ',150.00)" alt="brown boxer mix dog">
+          <h4>Angus</h4>
+          <h5>Cost to Adopt: $150.00</h5>
+          <p>Angus is a Boxer. He is 2 years old, neutered, microchipped, vaccinated, house/leash trained and ready for a new forever home.
+          </p>
+            <button type="button" onclick="feeTotal(150.00)">Adopt</button>
+        </article>
+        <article>
+          <img src="./images/violet-card.jpg" onclick="dogInfo('Violet ','Chocolate lab dachsund mix ',200.00)"  alt="brown blue eyed puppy">
+          <h4>Violet</h4>
+          <h5>Cost to Adopt: $200.00</h5>
+          <p>Violet is chocolate lab dachsund mix. She is 6 weeks old neutered, microchipped, vaccinated and ready for a new forever home.
+          </p>
+            <button type="button" onclick="feeTotal(200.00)">Adopt</button>
+      </article>
```

Comment:
be careful with your indenting and try to not have big blocks of empty space. it's just good formatting and good "penmanship"